greedy_method/greedy_coin_change.py

```diff
@@ -0,0 +1,48 @@
+"""
+Test cases:
+find_minimum_change(987)
+500 100 100 100 100 50 20 10 50 2
+find_minimum_change(500)
+500
+find_minimum_change(0)
+The total value cannot be zero or negetive
+find_minimum_change(-96)
+The total value cannot be zero or negetive
+find_minimum_change(56)
+50 5 1
+"""
+
+
+def find_minimum_change(V):
```

Comment:
Variable names are lowercase, and one-letter variable names are old school. Also, type hints should be added:

```python
def find_minimum_change(value: int) -> list:
    ...
```

Also, algorithms should not print values, they should return them. In this case, I suggest the function returns a list of integers, where each integer is a denomination.